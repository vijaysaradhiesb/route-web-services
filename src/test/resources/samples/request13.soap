<soapenv:Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:ws="http://ws.integclaims.itf.com">
   <soapenv:Header/>
   <soapenv:Body>
      <ws:documentAttachDocument soapenv:encodingStyle="http://schemas.xmlsoap.org/soap/encoding/">
         <username xsi:type="xsd:string">webservice</username>
         <password xsi:type="xsd:string">password</password>
         <documentId xsi:type="xsd:long">0</documentId>
         <claimId xsi:type="xsd:long">316179</claimId>
         <claimNumber xsi:type="xsd:string">16/0019013</claimNumber>
         <externalClaimReference xsi:type="xsd:string">CR4</externalClaimReference>
         <documentType xsi:type="xsd:string">DOCUMENT</documentType>
         <documentSubType xsi:type="xsd:string">STATEMENT_OF_FACT</documentSubType>
         <description xsi:type="xsd:string">vijay-test-claim-document</description>
         <relatedPartyId xsi:type="xsd:long">1571304</relatedPartyId>
         <documentFileName xsi:type="xsd:string">vijay-test-claim.tif</documentFileName>
         <attachedDocument xsi:type="xsd:base64Binary">QUJDCg==</attachedDocument>
         <senderEmailAddress xsi:type="xsd:string">integ.saradhi@integ.com</senderEmailAddress>
      </ws:documentAttachDocument>
   </soapenv:Body>
</soapenv:Envelope>