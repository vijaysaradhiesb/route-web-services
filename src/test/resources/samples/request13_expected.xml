<?xml version="1.0" encoding="UTF-8" standalone="yes"?><GenericWebServiceMessage xmlns="http://generic.message.integclaims.integ.com"><name>documentAttachDocument</name><keyValuePair><key>username</key><valueType>STRING</valueType><stringValue>webservice</stringValue></keyValuePair><keyValuePair><key>password</key><valueType>STRING</valueType><stringValue>password</stringValue></keyValuePair><keyValuePair><key>documentId</key><valueType>LONG</valueType><longValue>0</longValue></keyValuePair><keyValuePair><key>claimId</key><valueType>LONG</valueType><longValue>316179</longValue></keyValuePair><keyValuePair><key>claimNumber</key><valueType>STRING</valueType><stringValue>16/0019013</stringValue></keyValuePair><keyValuePair><key>externalClaimReference</key><valueType>STRING</valueType><stringValue>CR4</stringValue></keyValuePair><keyValuePair><key>documentType</key><valueType>STRING</valueType><stringValue>DOCUMENT</stringValue></keyValuePair><keyValuePair><key>documentSubType</key><valueType>STRING</valueType><stringValue>STATEMENT_OF_FACT</stringValue></keyValuePair><keyValuePair><key>description</key><valueType>STRING</valueType><stringValue>vijay-test-claim-document</stringValue></keyValuePair><keyValuePair><key>relatedPartyId</key><valueType>LONG</valueType><longValue>1571304</longValue></keyValuePair><keyValuePair><key>documentFileName</key><valueType>STRING</valueType><stringValue>vijay-test-claim.tif</stringValue></keyValuePair><keyValuePair><key>attachedDocument</key><valueType>BYTE_ARRAY</valueType><binaryValue>QUJDCg==</binaryValue></keyValuePair><keyValuePair><key>senderEmailAddress</key><valueType>STRING</valueType><stringValue>vijay.saradhi@huibio.com</stringValue></keyValuePair></GenericWebServiceMessage>